libwebrtc-hw/vpl_video_encoder.cpp

```diff
@@ -0,0 +1,512 @@
+#include "vpl_video_encoder.h"
+
+#include <chrono>
+#include <mutex>
+
+// WebRTC
+#include <modules/video_coding/codecs/h264/include/h264.h>
+#include <modules/video_coding/include/video_codec_interface.h>
+#include <modules/video_coding/include/video_error_codes.h>
+#include <rtc_base/logging.h>
+#include <rtc_base/synchronization/mutex.h>
+
+// Intel VPL
+#include <fcntl.h>
+#include <mfxdispatcher.h>
+#include <mfxvideo++.h>
+#include <mfxvideo.h>
+#include <mfxvp8.h>
```

Comment:
I think the proper include path is to the parent directory, no?
At least that's how it's installed under `${CMAKE_INSTALL_PREFIX}/include/vpl/*.h`

```suggestion
#include <vpl/mfxvp8.h>
```

Reply:
The current approach is also working because we expose the include path to these headers, should I prefix it with vpl for clarity?

Reply:
It's ok for now. But once you properly install vpl, you need to fix the include paths.